prime-dotnet-webapi/Controllers/EnrolleesAccessTermsController.cs

```diff
@@ -96,7 +96,7 @@ public async Task<ActionResult<AccessTerm>> GetAccessTerm(int enrolleeId, int ac
                 return Forbid();
             }
 
-            AccessTerm accessTerm = await _accessTermService.GetEnrolleeAccessTermAsync(enrolleeId, accessTermId, true);
+            Agreement accessTerm = await _accessTermService.GetEnrolleeAccessTermAsync(enrolleeId, accessTermId, true);
```

Comment:
Could the variable be renamed as well?

Reply:
I was just doing the minimum database changes for org agreements required to allow others to start working on this as well. Still lots to change in the code itself

Reply:
fair

Reply:
If we're going to do the minimum to unblock we should keep a list of what needs to found and renamed like local variables (accessTerm) and services (accessTermService) so we don't forget (or overlook) them later in the sprint.

Reply:
Added to the backend subtask